client/lib/application/src/approval-form/approval-form.jsx

```diff
@@ -133,6 +133,11 @@ class ApprovalForm extends React.Component {
         const {applicationId, versionId, application, approvalTypes, approvals, editable, userInfos} = this.props,
         LINK_PARAMS = {applicationId, versionId};
         return <div className='approvalForm'>
+                    <div style={{backgroundColor: '#f08532'}}>
```

Comment:
Would make sense but this is only there for the next few weeks. So I consider this to be ok.